clion-plugin/src/test/kotlin/com/github/vol0n/utbotcppclion/MyPluginTest.kt

```diff
@@ -0,0 +1,31 @@
+package com.github.vol0n.utbotcppclion
+
```

Comment:
what does it test?

Reply:
It is a template for writing tests in IntelliJ Platform, so it tests nothing.  Right now I am working on actual tests